overlay/overlay.nix

```diff
@@ -98,6 +105,8 @@ in
       ];
     });
 
+    # FIXME: pd-mapper reads from the firmware directory.
+    compressFirmwareXz = super.lib.id;
```

Comment:
TODO:

 - [ ] Verify if this is actually needed; maybe only apply in SDM845 family?